driver-async/src/test/functional/com/mongodb/async/client/ClientSideEncryptionTest.java

```diff
@@ -204,9 +204,10 @@ public void setUp() {
             if (extraOptionsDocument.containsKey("mongocryptdURI")) {
                 extraOptions.put("mongocryptdURI", extraOptionsDocument.getString("mongocryptdURI").getValue());
             }
-            if (System.getProperty("org.mongodb.test.mongocryptdPath") != null) {
-                extraOptions.put("mongocryptdPath", System.getProperty("org.mongodb.test.mongocryptdPath"));
-            }
+        }
+
+        if (System.getProperty("org.mongodb.test.mongocryptdSpawnPath") != null) {
```

Comment:
Change the name of the property to match the option key, and moved it out of the conditional because it should apply no matter what.

Reply:
Ack